ci/logging_tests_local.yml

```diff
@@ -0,0 +1,287 @@
+  vars_files:
+    - vars/common.yml
+  tasks:
+    - name: Create log dir
+      ansible.builtin.file:
+        path: "{{ logs_dir }}/logging_local"
+        state: directory
+        mode: "0755"
+
+#    # This needs to be updated; what do we use elsewhere?
+#    # Elsewhere, we assume that we're already logged in
+#    - name: "Log into OCP"
+#      ansible.builtin.shell:
+#        cmd: |
+#          oc login -u kubeadmin -p "12345678" https://api.crc.testing:6443
+#      failed_when: false
+#      changed_when: false
+#
+#    # I don't think we need this, Zuul __should__ have taken care of this already
+#    - name: "Create id rsa file for ctlplane access"
+#      ansible.builtin.shell:
+#        cmd: |
+#          oc extract -n openstack secrets/dataplane-ansible-ssh-private-key-secret --to=- | grep -v "ssh\-rsa" > "{{ id_rsa_path }}"
+#      register: rsa_results
+#      failed_when: rsa_results.rc != 0
+#      changed_when: false
+#
+#    # (efoley): Once again, I don't think this is necessary; if it's not needed for zuul, because it's already done, but is needed before running this on other systems, then the README should be updated to reflect this;
+#    # A troubleshooting section can be added too so users can identify when they are missing these configuration steps
+#     - name: "Change files permissions"
+#       ansible.builtin.shell:
+#         cmd: |
+#           chmod 600 "{{ id_rsa_path }}"
+#       changed_when: false  
+
+- name: "Verify logging projects, endpoints, credentials, nodes, pods, services, manifests and subscriptions"
+  hosts: controller
+  gather_facts: no
+  ignore_errors: true
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  vars:
+    proj_test_id: "RHOSO-12668"
+    proj_list:
+      - openshift-openstack-infra
+      - openshift       
+      - openstack-operators
+      - openshift-logging
+    endpoint_test_id: "RHOSO-12682"
+    endpoint_list:
+      - [nova,compute,public]
+      - [nova,compute,internal]
+      - [placement,placement,public]
+      - [placement,placement,internal]
+      - [swift,object-store,public]
+      - [swift,object-store,internal]
+      - [cinderv3,volumev3,public]
+      - [cinderv3,volumev3,internal]
+      - [barbican,key-manager,public]
+      - [barbican,key-manager,internal]
+      - [keystone,identity,public]
+      - [keystone,identity,internal]
+      - [glance,image,public]
+      - [glance,image,internal]
+      - [neutron,network,public]
+      - [neutron,network,internal]
+
+    cred_test_id: "RHOSO-12670"
+    cred_list:
+      - alertingrules.loki.grafana.com
+      - lokistacks.loki.grafana.com
+      - recordingrules.loki.grafana.com
+      - rulerconfigs.loki.grafana.com
+
+    node_test_id: "RHOSO-12671"
+    node_list:
+      - edpm-compute-0
+      - edpm-compute-1
+      - crc
+
+    pod_test_id: "RHOS0-12672"
+    pod_status_str: "Running"      
+    pod_nspace: openstack-operators
```

Comment:
In the qe_common role, the corresponding tasks expect "ns" to be defined, not "pod_namespace"